c/src/ml-api-service-agent-client.c

```diff
@@ -143,9 +143,8 @@ ml_service_get_pipeline (const char *name, char **pipeline_desc)
         "The parameter, 'name' is NULL, It should be a valid string");
   }
 
-  if (!pipeline_desc) {
-    _ml_error_report_return (ML_ERROR_INVALID_PARAMETER,
-        "The parameter 'pipeline_desc'. It should be a valid char**");
+  if (pipeline_desc) {
```

Comment:
ml_agent_dbus_interface_pipeline_get_description does the null check of pipeline_desc?

Reply:
@myungjoo

```ml_agent_dbus_interface_pipeline_get_description``` does, and it would be a problem only in the case of ```ml_agent_dbus_interface_pipeline_get_description(..., NULL, ...)```.